.babelrc

```diff
@@ -0,0 +1,11 @@
+{
+  "presets": [
+    "es2015",
+    "stage-0"
+  ],
+  "env": {
+    "test": {
+      "plugins": [ "istanbul" ]
+    }
+  }
+}
```

Comment:
Is this file used? I don't think so, so let's remove it.

Reply:
The entire .babelrc?